setupext.py

```diff
@@ -765,7 +767,7 @@ def include_dirs_hook():
         ext.include_dirs.append(numpy.get_include())
         if not has_include_file(
                 ext.include_dirs, os.path.join("numpy", "arrayobject.h")):
-            warnings.warn(
+            _log.warning(
```

Comment:
User action required -> `_warn_extenal`.

Reply:
This doesn't live in the matplotlib package so "external" will point to this very file; showing the source won't help.